client/landing/stepper/declarative-flow/flows/ai-site-builder/ai-site-builder.ts

```diff
@@ -1,4 +1,3 @@
-import config from '@automattic/calypso-config';
```

Comment:
For this files changes we're just rolling back to the current trunk version of the file.